internal/services/devtestlabs/dev_test_virtual_network_resource.go

```diff
@@ -96,6 +97,37 @@ func resourceArmDevTestVirtualNetwork() *pluginsdk.Resource {
 							Default:      string(virtualnetworks.UsagePermissionTypeAllow),
 							ValidateFunc: validate.DevTestVirtualNetworkUsagePermissionType(),
 						},
+
+						"shared_public_ip_address_configuration": {
```

Comment:
`configuration` is redundant here, this should be renamed to 
```suggestion
						"shared_public_ip_address": {
```

Reply:
Will do.